ops/ruby/devops-framework/DEVOPS_FRAMEWORK.md

```diff
@@ -0,0 +1,78 @@
+# DevOps Framework
```

Comment:
nit: Should this file just be called README.md? the file path looks a bit redundant